blog/views.py

```diff
@@ -32,7 +32,7 @@ def post_new(request):
             post_data = serializers.serialize('json', [post, ])
             return JsonResponse({'post_data': post_data}, status=HTTPStatus.CREATED)
         else:
-            msg = {"message": "잘못된 입력입니다."}
+            msg = {"message": "잘못된 형식입니다."}
             return JsonResponse(msg, status=HTTPStatus.BAD_REQUEST)
```

Comment:
여기도 역시 inline 해도 되겠네요~

```suggestion
            return JsonResponse({"message": "잘못된 형식입니다."}, status=HTTPStatus.BAD_REQUEST)
```